src/remote.sh

```diff
@@ -48,13 +48,21 @@ function cmd_remotely()
   # Set default values if not specified
   remote=${remote:-"localhost"}
   port=${port:-"22"}
-  user=${user:-"root"}
+  user=${user:-$(whoami)}
   flag=${flag:-"HIGHLIGHT_CMD"}
 
-  composed_cmd="ssh -p $port $user@$remote \"$command\""
+  if [[ -v configurations['ssh_configfile'] ]]; then
+    composed_cmd="ssh -F ${configurations['ssh_configfile']} ${configurations['hostname']}"
```

Comment:
The configuration variable `hostname` is used without checking if it exists! This could be checked and warned or solved at `kw_config_loader`. I think not handling this error might be user-unfriendly.